src/tmpfiles/tmpfiles.c

```diff
@@ -2912,16 +3065,35 @@ static int parse_line(
 
         if (!empty_or_dash(age)) {
                 const char *a = age;
+                _cleanup_free_ char *seconds = NULL, *age_by = NULL;
 
                 if (*a == '~') {
                         i.keep_first_level = true;
                         a++;
                 }
 
+                /* Format: "age-by:age"; where age-by is "[abcmABM]+". */
+                r = split_pair(a, ":", &age_by, &seconds);
+                if (r == -ENOMEM)
+                        return log_oom();
```

Comment:
please propagate all errors here, except for EINVAL. i.e. when handling errors, there should always be a catchall for unexpected errors. it's only EINVAL that we need to take care of explicitly

Reply:
ACK.